docs/analyzer/checker_and_analyzer_configuration.md

```diff
@@ -1,7 +1,33 @@
 # Configure Clang Static Analyzer and checkers
 
-Analyzer configuration can be done through the `--saargs` analysis option which
-forwards arguments without modification to the Clang Static Analyzer:
+## Analyzer Configuration <a name="analyzer-configuration"></a>
+
+The analysis can be configured using analyzer wide configuration parameters. These parameters may have effect on the whole analysis, affecting the result of all checkers.
```

Comment:
Please break long lines in the markdown files too. The preferred max-length of each line should be 80 characters.

Note: I created an issue to introduce a linter for markdown files but it's not ready yet.